frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -225,13 +224,9 @@ class Sidebar extends PureComponent<SidebarProps, State> {
             bottom: false,
             left: false,
           }}
+          handleStyles={{ right: { width: "8px", right: "-6px" } }}
           handleComponent={{
-            right: (
-              <>
-                <StyledResizeHandleMargin />
-                <StyledResizeHandle onClick={this.resetSidebarWidth} />
-              </>
-            ),
+            right: <StyledResizeHandle onClick={this.resetSidebarWidth} />,
```

Comment:
The `StyledResizeHandle` passed to the `handleComponent` prop actually renders as a child to re-resizeable's right-side handle component, so it was necessary to also adjust the parent styling through `handleStyles` prop.